### ISSUE TITLE
installation/setup: run choco once to initialize the config file

### ISSUE THREAD

**Issue by KirillOsenkov** [chocolatey/choco#1401]
### What You Are Seeing?

First time after installing Chocolatey as admin, I start a non-admin command prompt, run `choco pack` and see this error:
![image](https://user-images.githubusercontent.com/679326/30461713-538db968-9975-11e7-9971-d89aaec25e5a.png)

### What is Expected?
It should work fine (whatever Chocolatey wants to write to ProgramData needs to be written at install time). Alternatively, consider saving the config file to LocalAppData (this doesn't need admin permissions to write, and is generally a better thing to do. Program Data goes to one place, configs, logs etc. go to another).

### Output Log
Unfortunately I didn't grab the output log other than the screenshot above. I tried a workaround (which is to run pack under admin, and that created the config file and I got unblocked).

**Comment by KirillOsenkov** [comment]
And here's the output:
```
This is try 1/3. Retrying after 300 milliseconds.
 Error converted to warning:
 Access to the path 'C:\ProgramData\chocolatey\config' is denied.
This is try 2/3. Retrying after 400 milliseconds.
 Error converted to warning:
 Access to the path 'C:\ProgramData\chocolatey\config' is denied.
Maximum tries of 3 reached. Throwing error.
Cannot create directory "C:\ProgramData\chocolatey\config". Error was:
System.UnauthorizedAccessException: Access to the path 'C:\ProgramData\chocolatey\config' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost)
   at System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.<>c__DisplayClass64.<create_directory>b__63()
   at chocolatey.infrastructure.tolerance.FaultTolerance.<>c__DisplayClass1.<retry>b__0()
   at chocolatey.infrastructure.tolerance.FaultTolerance.retry[T](Int32 numberOfTries, Func`1 function, Int32 waitDurationMilliseconds, Int32 increaseRetryByMilliseconds, Boolean isSilent)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.create_directory(String directoryPath)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.create_directory_if_not_exists(String directoryPath, Boolean ignoreError)
Chocolatey had an error occur:
System.UnauthorizedAccessException: Access to the path 'C:\ProgramData\chocolatey\config' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost)
   at System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.<>c__DisplayClass64.<create_directory>b__63()
   at chocolatey.infrastructure.tolerance.FaultTolerance.<>c__DisplayClass1.<retry>b__0()
   at chocolatey.infrastructure.tolerance.FaultTolerance.retry[T](Int32 numberOfTries, Func`1 function, Int32 waitDurationMilliseconds, Int32 increaseRetryByMilliseconds, Boolean isSilent)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.create_directory(String directoryPath)
   at chocolatey.infrastructure.filesystem.DotNetFileSystem.create_directory_if_not_exists(String directoryPath, Boolean ignoreError)
   at chocolatey.infrastructure.extractors.AssemblyFileExtractor.extract_text_file_from_assembly(IFileSystem fileSystem, IAssembly assembly, String manifestLocation, String filePath, Boolean overwriteExisting)
   at chocolatey.infrastructure.app.builders.ConfigurationBuilder.get_config_file_settings(IFileSystem fileSystem, IXmlService xmlService)
   at chocolatey.infrastructure.app.builders.ConfigurationBuilder.set_up_configuration(IList`1 args, ChocolateyConfiguration config, Container container, ChocolateyLicense license, Action`1 notifyWarnLoggingAction)
   at chocolatey.console.Program.Main(String[] args)
```

**Comment by ferventcoder** [comment]
@KirillOsenkov thanks for adding this. Adding to backlog.

**Comment by tiget** [comment]
Could I take this on?

**Comment by ferventcoder** [comment]
Merged into stable at https://github.com/chocolatey/choco/commit/3cb969b92730ddc418b9b258b245c90969435d69 - will be released in v0.10.9.

**Comment by daihieptn97** [comment]
run PowerShell with admin rights
